WorkloadManagementSystem/Service/WMSUtilities.py

```diff
@@ -78,8 +79,8 @@ def getARCPilotOutput( proxy, pilotRef ):
   myce = pilotRef.split(":")[1].strip("/")
   gridEnv = getGridEnv()
   mySite = getCESiteMapping()['Value'][myce]
-  WorkDB = gConfig.getValue(os.path.join('Resources/Sites/LCG', mySite, 'CEs', myce, 'JobListFile'))
-  myWorkDB = os.path.join("/opt/dirac/runit/WorkloadManagement/SiteDirector-RAL2", WorkDB)
+  WorkDB = gConfig.getValue(cfgPath['Resources/Sites/LCG', mySite, 'CEs', myce, 'JobListFile'])
+  myWorkDB = os.path.join("/opt/dirac/runit/WorkloadManagement/SiteDirector-RAL", WorkDB)
```

Comment:
It was a sort of hack before, and this is not better. Can you make something generic? Do you need this specific directory?

Also, minor comments: 
- variables should be in camelCase with first letter lowercase, so use "workDB" not "WorkDB".
- there's a "print" few lines below, please remove it
- are lines 103-107 needed?